finetune/util/huggingface_interface.py

```diff
@@ -44,12 +45,13 @@ def finetune_model_from_huggingface(
     hf_tokenizer,
     hf_config,
     weights_replacement,
+    include_bos_eos=True,
 ):
     weights_url = archive_map[pretrained_weights]
     hf_tokenizer_instance = hf_tokenizer.from_pretrained(pretrained_weights)
     hf_config_instance = hf_config.from_pretrained(pretrained_weights)
 
-    def finetune_featurizer(X, encoder, config, train=False, reuse=None, **kwargs):
+    def finetune_featurizer(X, encoder, config, train=False, reuse=None, targets=None, lengths=None, **kwargs):
```

Comment:
Should we use this on other models? I definite wrote this code once before but it seems it never got merged

Reply:
Targets doesn't seem to be required any longer.  Do you mean the lengths argument?

Reply:
Yep, lengths! I really meant to put this comment on the block that generates the lengths

Reply:
Yeah I'm down for that change.